resources/views/pets.blade.php

```diff
@@ -0,0 +1,439 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Pets!</title>
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700&display=swap" rel="stylesheet">
+
+    <!-- Styles -->
+    <style>
+        /*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
probably worth linking from a CDN so you don't clutter your view, but adding this is a nice touch

Reply:
I think I just repurposed Laravel's existing view template files, so I can only take credit for carrying forward the convention they established 😁